### PR TITLE
Use themable symbol instead of svg source

### PR DESCRIPTION
**Pull request by jonesbusy**
Use themable symbol instead of svg source

Before

before_jcasc1

After

Default

after_jcasc1

Dark

after_jcasc2

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/configuration-as-code.svg?color=blue)](https://plugins.jenkins.io/configuration-as-code)
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
 
-<img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
+<img src="plugin/src/main/resources/images/symbols/logo.svg" width="192">
```
Comment (timja) [inline README.md:10]: I think we should keep colour in the README it doesn't look that great as-is on dark at least:

image

File: plugin/pom.xml [hunk 2]
```diff
@@ -34,7 +34,7 @@
     -->
     <hpi.compatibleSinceVersion>1.28</hpi.compatibleSinceVersion>
     <hpi.pluginChangelogUrl>https://github.com/jenkinsci/configuration-as-code-plugin/releases</hpi.pluginChangelogUrl>
-    <hpi.pluginLogoUrl>https://raw.githubusercontent.com/jenkinsci/configuration-as-code-plugin/master/plugin/src/main/webapp/img/logo-head.svg</hpi.pluginLogoUrl>
+    <hpi.pluginLogoUrl>https://raw.githubusercontent.com/jenkinsci/configuration-as-code-plugin/master/plugin/src/main/resources/images/symbols/logo.svg</hpi.pluginLogoUrl>
```
Comment (timja) [inline plugin/pom.xml:37]: interesting I don't think this is actually used anywhere tbh

File: plugin/src/main/resources/images/symbols/logo.svg
Comment (timja) [inline plugin/src/main/resources/images/symbols/logo.svg:1]: Default theme looks quite nice, does dark theme look a bit funny to you?

thoughts @janfaracik ?

**Review by timja — COMMENTED** [review]
Hey thanks for the PR, here's my thoughts atm:

File: plugin/src/main/resources/images/symbols/logo.svg
Comment (janfaracik) [reply plugin/src/main/resources/images/symbols/logo.svg:1]: The 'About Jenkins' icon looks a bit off too to be fair in dark mode, think this is fine as its consistent with that.

File: plugin/pom.xml (hunk 2 above)
Comment (jonesbusy) [reply plugin/pom.xml:37]: No idea. It's the first time I see a pom with `hpi.pluginLogoUrl`

Comment (timja) [reply plugin/pom.xml:37]: 2 plugins have it according to:
https://github.com/search?q=org%3Ajenkinsci%20hpi.pluginLogoUrl&type=code

but it appears to be unused anywhere, I think it was unfinished work to maybe show in plugin manager or plugins.jenkins.io

File: README.md (hunk 1 above)
Comment (jonesbusy) [reply README.md:10]: Sure, the `currentColor` is only honered if the SVG is added inline. I can keep the color logo only for the doc

File: plugin/src/main/resources/images/symbols/logo.svg
Comment (jonesbusy) [reply plugin/src/main/resources/images/symbols/logo.svg:1]: Yea, it was my feeling too. No idea the reason. It's like the stroke is eavier in dark mode

about_default

about_dark

**Comment by jonesbusy** [comment]
I have restored the original logo for the readme only

**Comment by jonesbusy** [comment]
https://github.com/jenkinsci/configuration-as-code-plugin/actions/runs/11919116816/job/33218127817#step:4:1

@timja Again a maven download falure here

**Comment by timja** [comment]
filed https://github.com/jenkins-infra/github-reusable-workflows/issues/40

**Pull request merged**